pkg/daemon/ceph/osd/volume.go

```diff
@@ -102,7 +105,24 @@ func (a *OsdAgent) configureCVDevices(context *clusterd.Context, devices *Device
 
 func (a *OsdAgent) initializeBlockPVC(context *clusterd.Context, devices *DeviceOsdMapping, lvBackedPV bool) (string, error) {
 	baseCommand := "stdbuf"
-	baseArgs := []string{"-oL", cephVolumeCmd, "lvm", "prepare"}
+	var baseArgs []string
+
+	// Create a specific log directory so that each prepare command will have its own log
+	// Only do this if nothing is present so that we don't override existing logs
+	cvLogDir = path.Join(cephLogDir, a.nodeName)
```

Comment:
Are these changes only specific to OSD on PVC? Should we have it for both `OSD on PVC` and `OSD on raw devices`?

Reply:
Yes, they only make sense when running on PVC. OSD on raw devices don't exist yet, they only exist here: https://github.com/rook/rook/pull/4435